contracts/modules/staking/StakingPool.sol

```diff
@@ -922,31 +922,61 @@ contract StakingPool is IStakingPool, Multicall {
 
       uint[] memory trancheCapacities = getTrancheCapacities(
         request.productId,
-        firstTrancheIdToUse,
-        MAX_ACTIVE_TRANCHES - startIndex, // count
+        _firstActiveTrancheId,
```

Comment:
what's the logic behind starting with _firstActiveTrancheId vs firstTrancheIdToUse

Reply:
- first active tranche id - exactly what it says, from all active tranches it has the lowest id
- first active to use - from all active ones, the first one that expires after the cover grace period, basically the first one we can use for the current cover